### PR TITLE
Youbora sessions documentation

### PR DESCRIPTION
**Pull request by jesuslc91**

### CODE REVIEW

File: docs/YouboraSessions.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Youbora sessions integration
+
+## To integrate youbora sessions follow this documentation:
+https://documentation.npaw.com/npaw-integration/docs/application-analytics-custom-events-data-collection
```
Comment (OrenMe) [inline docs/YouboraSessions.md:4]: need to add part on how to pass plugin instance to player on the plugin config

Comment (jesuslc91) [reply docs/YouboraSessions.md:4]: I need plugin config creation sample to add that part

Comment (OrenMe) [reply docs/YouboraSessions.md:4]: https://github.com/kaltura/kaltura-player-android-samples/blob/develop/AdvancedSamples/Youbora/app/src/main/java/com/kaltura/playkit/samples/youbora/MainActivity.kt#L287

Comment (jesuslc91) [reply docs/YouboraSessions.md:4]: Actually we don't need the plugin instance, only the fastdataconfig on the plugin options, here: https://github.com/kaltura/playkit-android-youbora/blob/bb1bf1fff5e46453150444ea3307587d71ff2dd7/youboraplugin/src/main/java/com/kaltura/playkit/plugins/youbora/pluginconfig/YouboraConfig.java#L303

**Review by OrenMe — APPROVED** [review]

**Pull request merged**